### PR TITLE
Meet/hotfix

### PR DESCRIPTION
**Pull request by meetmangukiya**
# Reviewers Checklist

- [ ] Appropriate logging is done.
- [ ] Appropriate error responses.
- [ ] Handle every possible exception.
- [ ] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [ ] See that 100% coverage is there.
- [ ] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

File: tests/labhub_test.py [hunk 1]
```diff
@@ -151,9 +158,12 @@ def test_assign_cmd(self):
                               'You\'ve been assigned to the issue')
 
         # no assignee, newcomer, no labels
+        self.mock_team.is_member.return_value = True
         mock_issue.labels = tuple()
+        mock_issue.assignees = tuple()
         testbot.assertCommand(cmd.format('coala', 'a', '23'),
-                              'not eligible to be assigned to this issue')
+                              'not eligible to be assigned to this issue',
+                                timeout=1000)
```
Comment (meetmangukiya) [inline tests/labhub_test.py]: remove this

File: plugins/labhub.py [hunk 2]
```diff
@@ -239,7 +247,13 @@ def newcomer_difficulty_level(user, iss):
             2. A newcomer asks for assignment to an issue with difficulty higher
                than low.
             """
-            if self.TEAMS[self.GH_ORG_NAME + ' newcomers'].is_member(user):
+            if (self.TEAMS[self.GH_ORG_NAME + ' newcomers'].is_member(user) and
```
Comment (jayvdb) [inline plugins/labhub.py]: this looks overly complicated, and wrong.

if someone is a maintainer, this `if` clause will be skipped, and then the `elif self.GH3_ORG.is_member(user):` block will be executed, which means this method will return `True`.  which doesnt sound right.  Have I misunderstood somehow?

Comment (meetmangukiya) [inline plugins/labhub.py]: Yes, isn't that expected, maintainers are eligible to be assigned to issues that are not labelled or are higher than low

Comment (jayvdb) [inline plugins/labhub.py]: Ah ok, then it needs to be renamed as `newcomer_difficulty_level` implies True means newcomer difficulty level issues are allowed.

Comment (meetmangukiya) [inline plugins/labhub.py]: I guess, since it checks for all statures, we rename it to just `difficulty_level`

Comment (meetmangukiya) [inline plugins/labhub.py]: Also, changed the docstring :+1:

**Comment by Udayan12167** [comment]
ack https://github.com/coala/corobo/pull/146/commits/c176a1c72b3cd956735d5c6b9288c47143250759

**Comment by Udayan12167** [comment]
ack https://github.com/coala/corobo/pull/146/commits/7115e6bb591c8c986dddf56d5010108da51724d7

**Comment by Udayan12167** [comment]
ack
https://github.com/coala/corobo/pull/146/commits/e535f99aa00afd8008828a8bb45534022fba7283

**Comment by Udayan12167** [comment]
ack https://github.com/coala/corobo/pull/146/commits/24e38cf5dd798c18b7214cfd631e23554db9a138

**Comment by Udayan12167** [comment]
ack https://github.com/coala/corobo/pull/146/commits/0ab7181dea98e3c386804a9b99a2f31e48d72e7c

**Comment by meetmangukiya** [comment]
ack 21091edc607a785380579b1a1e0613cfaab406c0 a9de37f1128318029a69af30cc7c6031f299cf92  32f0421fa165e4d8600c67adefa3a282428ba8fd  d739209bc7be4583e0df1711e771217963262dbd  83fa29a1da8bcc1e026c5568b9c7881458fa1301

**Comment by meetmangukiya** [comment]
@rultor merge

**Pull request merged**